sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaFormat.cs

```diff
@@ -11,6 +11,10 @@ namespace Azure.Data.SchemaRegistry
     [CodeGenModel("SchemaFormat")]
     public readonly partial struct SchemaFormat
     {
+        private const string AvroValue = "Avro";
```

Comment:
Why did we need to add these?

Reply:
I added these because the generator was setting AvroValue as "application/json; serialization=Avro" (etc) and it was breaking the SchemaFormat Tests, to me it seemed like the value should be "Avro"? But if it's not breaking/not an issue, I can switch them back

Reply:
Ah I see. I didn't remember that the ToString used these values. You are right we need to add these back.